opentracing/ext/tags.go

```diff
@@ -9,44 +9,38 @@ var (
 	// the other side/service in a peer-to-peer communications, like an RPC call.
 
 	// PeerService records the service name of the peer
-	PeerService = &stringTag{"peer.service"}
+	PeerService = stringTag("peer.service")
```

Comment:
Not related to your change.. This is sortof like the "sa" and "ca" annotations in zipkin. https://github.com/openzipkin/zipkin/blob/master/zipkin-thrift/src/main/thrift/com/twitter/zipkin/zipkinCore.thrift#L144